<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_login_window</class>
 <widget class="QDialog" name="Dialog_login_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>411</width>
    <height>301</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ZoomPlusPlus</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_meeting_id">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Meeting ID</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_meeting_id">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="placeholderText">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_password">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Password</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_password"/>
     </item>
     <item>
      <widget class="QLabel" name="label_url">
       <property name="text">
        <string>Url</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_url">
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>with meeting id, url is optional</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <!-- <item row="1" column="0">
    <widget class="QPushButton" name="pushButton_login">
     <property name="text">
      <string>Join Metting</string>
     </property>
    </widget>
   </item> -->
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
